src/w2utils.js

```diff
@@ -1332,7 +1332,7 @@ w2utils.event = {
             '<div id="w2ui-overlay'+ name +'" style="display: none; left: 0px; top: 0px; '+ options.overlayStyle +'"'+
             '        class="w2ui-reset w2ui-overlay '+ ($(this).parents('.w2ui-popup, .w2ui-overlay-popup').length > 0 ? 'w2ui-overlay-popup' : '') +'">'+
             '    <style></style>'+
-            '    <div style="'+ options.style +'" class="'+ options['class'] +'"></div>'+
+            '    <div style="min-width: 100%; '+ options.style +'" class="'+ options['class'] +'"></div>'+
```

Comment:
@vitmalina know I remember me why I added this style property. The real bug is a wrong inline style width in the inner div. The simplest solution was the adding min-width: 100% to this inner div. Look at the pictures below.

![Bug](http://fs2.directupload.net/images/150719/6a8ni58w.png)

![DOM](http://fs2.directupload.net/images/150719/xpxrkjtb.png)